### PR TITLE
Added Error Dropdown Toast and implement it on Password Recovery Modal

### PR DESCRIPTION
**Pull request by amanalili**
Screen Shot 2021-01-06 at 12 33 23 AM

### CODE REVIEW

File: packages/edge-login-ui-rn/src/types/Theme.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ export type Theme = {|
 
   pressedOpacity: number,
   fontFamily: string,
+  fontWeightBold: string,
+
+  // Icons
+  icon: string,
```
Comment (swansontec) [inline packages/edge-login-ui-rn/src/types/Theme.js:12]: The `icon` and `warningText` styles are wrong here. They should be `dropdownWarning`, `dropdownError`, and `dropdownText` which would be used for both the icons and the text.

File: packages/edge-login-ui-rn/src/util/getHeaderHeight.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// @flow
+import { Platform } from 'react-native'
+
+/**
+ * Calculates the height of the header (where the back button lives).
+ */
+export function getHeaderHeight() {
```
Comment (swansontec) [inline packages/edge-login-ui-rn/src/util/getHeaderHeight.js:7]: Do we really need to cover this area with the dropdown? What if the dropdown doesn't have a minHeight? I know the react-native-airship itself handles the notch thing.

**Review by swansontec — CHANGES_REQUESTED** [review]
I have one change needed and one question for you.

File: packages/edge-login-ui-rn/src/types/Theme.js (hunk 1 above)
Comment (amanalili) [reply packages/edge-login-ui-rn/src/types/Theme.js:12]: Fixed

File: packages/edge-login-ui-rn/src/util/getHeaderHeight.js (hunk 2 above)
Comment (amanalili) [reply packages/edge-login-ui-rn/src/util/getHeaderHeight.js:7]: Yeah I guess it looks fine without the getHeaderHeight. Thats just the old one I copy so no idea what its really used for. Fixed

**Review by swansontec — APPROVED** [review]

**Pull request merged**